src/mono/browser/runtime/memory.ts

```diff
@@ -447,8 +447,8 @@ export function copyBytes (srcPtr: VoidPtr, dstPtr: VoidPtr, bytes: number): voi
 // on non-MT build, this will be a no-op trimmed by rollup
 export function receiveWorkerHeapViews () {
     if (!WasmEnableThreads) return;
-    const memory = runtimeHelpers.getMemory();
-    if (memory.buffer !== Module.HEAPU8.buffer) {
+    const wasmMemory = runtimeHelpers.getMemory();
+    if (wasmMemory.buffer !== Module.HEAPU8.buffer) {
```

Comment:
What's the reason of using `updateMemoryViewes` here, not `forceThreadMemoryViewRefresh` that differ with calling `wasmMemory.grow(0);`? We don't need to propagate the knowledge about the update between threads?

Reply:
I assume that grow(0) is relatively expensive.
But the `receiveWorkerHeapViews` is called from too many places.